source/reference/minio-server/settings/storage-class.rst

```diff
@@ -0,0 +1,76 @@
+.. _minio-server-envvar-storage-class:
+.. _minio-ec-storage-class:
+
+======================
+Storage Class Settings
```

Comment:
Hm - so we don't talk about Erasure Coding or Parity at all in the title.

Would

Erasure Code (Storage Class) Settings 

Be too long?

Reply:
Changed the title to just "Erasure Code Settings" then worked on the page overview to clarify a bit more about how these settings work with links to relevant other topics that were recently expanded or added.